src/apps/inventory/views.py

```diff
@@ -76,7 +81,7 @@ def get_queryset(self):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context["filter"] = self.filterset_class(self.request.GET)
+        context["product_image"] = ProductImage.objects.filter(is_active=True).first()
```

Comment:
I still concerned about do we really need product_image in context. As I can't find usages for it.

Reply:
you're right. I have already cleaned my code from it everywhere. Here I missed it.